packages/@o3r/core/schematics/index.it.spec.ts

```diff
@@ -28,7 +24,7 @@ describe('new otter application', () => {
       execAppOptions.cwd = appFolderPath;
     });
     test('should build empty app', () => {
-      packageManagerExec(`ng add --skip-confirmation @o3r/core@${o3rVersion} --preset=cms`, execAppOptions);
+      packageManagerExec(`ng add --skip-confirmation @o3r/core@${o3rVersion} --preset=all`, execAppOptions);
```

Comment:
shouldn't both work?

Reply:
Nope, the Analytics is not part of the CMS preset and the following test requires the Analytics package to be installed.
(It used to work because relying on the `--enable-analytics` deprecated option